### PR TITLE
Remove cross entropy sigmoid loss as deprecated

### PR DESCRIPTION
**Pull request by adekusar-drl**
### Summary
Remove `CrossEntropySigmoidLoss` as deprecated in the version 0.4.

### CODE REVIEW

**Comment by woodsp-ibm** [comment]
This should have a release note for the removal.

**Comment by adekusar-drl** [comment]
> This should have a release note for the removal.

Added a reno file.

**Review by woodsp-ibm — APPROVED** [review]
